drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -479,27 +506,6 @@ local function do_configure(driver, device)
   else
     device.log.warn_with({hub_logs=true}, "Device does not support thermostat cluster")
   end
-
-  --Query setpoint limits if needed
-  local setpoint_limit_read = im.InteractionRequest(im.InteractionRequest.RequestType.READ, {})
```

Comment:
Can you help remind me why we are including these values in subscriptions instead of manually sending a read request? It was due to the issue where too many read attributes would be queued at once and the interaction queue would be full, right? I would include an explanation of this reasoning in your confluence page so we don't forget

Reply:
That's correct. I added a paragraph in the middle of the [Implementation](https://smartthings.atlassian.net/wiki/spaces/CHIP/pages/3110142325/Dynamic+Capability+Constraints#Implementation) section of the confluence page with an explanation. Here is what I added for reference: "However, this approach resulted in too many read attributes queuing at the same time occassionally, and as a result the interaction queue would fill up resulting in an exception. The drivers were later updated to remove the read requests and rather use subscriptions for these attributes in order to take advantage of the existing optimizations for subscriptions to avoid this issue."